packages/angular_devkit/schematics/README.md

```diff
@@ -62,37 +62,60 @@ A `Source` is a generator of `Tree`; it creates a root tree from nothing. A `Rul
 
 It is impossible to create files using a `FileOperator`.
 
+## Provided Operators
+The Schematics library provides multiple `Operator` factories by default that cover basic use cases:
+
 | FileOperator | Description |
 |---|---|
 | `contentTemplate<T>(options: T)` | Apply a content template (see the Template section) |
 | `pathTemplate<T>(options: T)` | Apply a path template (see the Template section) |
 
 ## Provided Sources
-The Schematics library provides multiple `Source` factories by default that cover basic use cases:
+The Schematics library additionally provides multiple `Source` factories by default:
 
 | Source | Description |
 |---|---|
-| `source(tree: Tree)` | Creates a source that returns the tree passed in argument. |
-| `empty()` | Creates a source that returns an empty tree. |
-| `apply(source: Source, rules: Rule[])` | Apply a list of rules to a source, and return the result. |
-| `url(url: string)` | Loads a list of files from a URL and returns a Tree with the files as `CreateAction` applied to an empty `Tree` |
+| `empty()` | Creates a source that returns an empty `Tree`. |
+| `source(tree: Tree)` | Creates a `Source` that returns the `Tree` passed in as argument. |
+| `url(url: string)` | Loads a list of files from the given URL and returns a `Tree` with the files as `CreateAction` applied to an empty `Tree`. |
+| `apply(source: Source, rules: Rule[])` | Apply a list of `Rule`s to a source, and return the resulting `Source`. |
```

Comment:
changed to order of the items to a (hopefully) more logical, "building on top of each other" one